examples/demo-apps/android/LlamaDemo/app/src/androidTest/java/com/example/executorchllamademo/PerfTest.java

```diff
@@ -24,33 +27,35 @@
 public class PerfTest implements LlamaCallback {
 
   private static final String RESOURCE_PATH = "/data/local/tmp/llama/";
-  private static final String MODEL_NAME = "xnnpack_llama2.pte";
   private static final String TOKENIZER_BIN = "tokenizer.bin";
 
-  // From https://github.com/pytorch/executorch/blob/main/examples/models/llama2/README.md
-  private static final Float EXPECTED_TPS = 10.0F;
-
   private final List<String> results = new ArrayList<>();
   private final List<Float> tokensPerSecond = new ArrayList<>();
 
   @Test
   public void testTokensPerSecond() {
-    String modelPath = RESOURCE_PATH + MODEL_NAME;
     String tokenizerPath = RESOURCE_PATH + TOKENIZER_BIN;
-    LlamaModule mModule = new LlamaModule(modelPath, tokenizerPath, 0.8f);
+    // Find out the model name
+    File directory = new File(RESOURCE_PATH);
+    Arrays.stream(directory.listFiles())
+        .filter(file -> file.getName().endsWith(".pte") || file.getName().endsWith(".pt"))
```

Comment:
We don't need `.pt` file, it's the suffix of the weights for stories110M

Reply:
I see, I can remove this, but it looks like something is not right where the https://gha-artifacts.s3.amazonaws.com/pytorch/executorch/10332366636/artifact/stories110M_xnnpack/model.zip contains only `stories110M.pt` and the tokenizer.  Should there be an exported `stories110M.pte` model there?  Maybe this explain why it fails to load.

Reply:
yeah, the fix is in https://github.com/pytorch/executorch/pull/4642

Reply:
So you merge the test spec fix and ignore the "failed to load model" issue

Reply:
> yeah, the fix is in #4642

Oh, got it, let me push a commit to fix the comments and land this PR